react/__tests__/components/Container.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+import Container from '../../Container'
+import { render } from 'enzyme'
+
+describe('<Container /> component', () => {
+  const renderComponent = customProps => {
+    const comp = <Container {...customProps}> Test </Container>
+
+    return render(comp)
+  }
+
+  const wrapper = renderComponent()
```

Comment:
I think it'd be better to run this function and create this wrapper on each test instead

Reply:
nice tip